docs/UpdaterConfig.md

```diff
@@ -84,14 +100,15 @@ predicted arrival and departure times for the remainder of the trip.
 <!-- stop-time-updater BEGIN -->
 <!-- NOTE! This section is auto-generated. Do not change, change doc in code instead. -->
 
-| Config Parameter                                                      |    Type   | Summary                                             |  Req./Opt. | Default Value        | Since |
-|-----------------------------------------------------------------------|:---------:|-----------------------------------------------------|:----------:|----------------------|:-----:|
-| type = "STOP_TIME_UPDATER"                                            |   `enum`  | The type of the updater.                            | *Required* |                      |   na  |
-| [backwardsDelayPropagationType](#u__5__backwardsDelayPropagationType) |   `enum`  | How backwards propagation should be handled.        | *Optional* | `"required-no-data"` |  2.2  |
-| feedId                                                                |  `string` | Which feed the updates apply to.                    | *Optional* |                      |   na  |
-| frequencySec                                                          | `integer` | How often the data should be downloaded in seconds. | *Optional* | `60`                 |   na  |
-| fuzzyTripMatching                                                     | `boolean` | If the trips should be matched fuzzily.             | *Optional* | `false`              |   na  |
-| url                                                                   |  `string` | The URL of the GTFS-RT resource.                    | *Required* |                      |   na  |
+| Config Parameter                                                      |       Type      | Summary                                                     |  Req./Opt. | Default Value        | Since |
+|-----------------------------------------------------------------------|:---------------:|-------------------------------------------------------------|:----------:|----------------------|:-----:|
+| type = "STOP_TIME_UPDATER"                                            |      `enum`     | The type of the updater.                                    | *Required* |                      |   na  |
+| [backwardsDelayPropagationType](#u__5__backwardsDelayPropagationType) |      `enum`     | How backwards propagation should be handled.                | *Optional* | `"required-no-data"` |  2.2  |
+| feedId                                                                |     `string`    | Which feed the updates apply to.                            | *Optional* |                      |   na  |
```

Comment:
`FeedId` and `type `type is both v1.5

Reply:
I've added a few more version in 88b4a6dd689e7b2d1361b891f40c18a45010fad4